modules/client-common/src/utils.ts

```diff
@@ -405,29 +423,46 @@ export async function* prepareGenericUpdate(
  */
 export function getNetwork(networkish: Networkish): Network {
   let network: Network | undefined;
-  for (const nw of ADDITIONAL_NETWORKS) {
+  try {
+    network = ethersGetNetwork(networkish);
+  } catch {
+  } finally {
     switch (typeof networkish) {
       case "string":
-        if (networkish === nw.name) {
-          network = nw;
-        }
-        break;
       case "number":
-        if (networkish === nw.chainId) {
-          network = nw;
+        const aragonNw = getNetworkByNameAliasOrChainId(networkish);
```

Comment:
nowadays there is no need to shorten variable names
```suggestion
        const aragonNetwork = getNetworkByNameAliasOrChainId(networkish);
```